roles/hammer_devel/tasks/hammer_install.yml

```diff
@@ -53,5 +53,11 @@
 - name: 'Install bundler'
   gem: name=bundler state=present
 
+- name: Set GEM_HOME
```

Comment:
This is a side effect of https://github.com/theforeman/forklift/pull/841 which switched to `rh-ruby25`. That's affected by https://bugzilla.redhat.com/show_bug.cgi?id=1613864#c5. See https://github.com/theforeman/puppet-katello_devel/pull/178#issuecomment-455719359 as well.

Perhaps we should do this as part of the SCL role to work around it. There we already deploy it in profile.d, maybe a better place? Maybe with that katello devel could even avoid using `.vendor` which in my experience led to a faster install.